README.md

```diff
@@ -1,2 +1,2 @@
-# daccs_node_python
+# DACCS python client
```

Comment:
I don't know if this is the best name but this is more human readable at least

Reply:
Thanks for correcting this!